lib/compress/zstd_compress_internal.h

```diff
@@ -191,9 +204,11 @@ struct ZSTD_CCtx_s {
     ZSTD_customMem customMem;
     size_t staticSize;
 
-    seqStore_t seqStore;    /* sequences storage ptrs */
-    ldmState_t ldmState;    /* long distance matching state */
-    rawSeq* ldmSequences;   /* Storage for the ldm output sequences */
+    seqStore_t seqStore;      /* sequences storage ptrs */
+    ldmState_t ldmState;      /* long distance matching state */
+    rawSeq* ldmSequences;     /* Storage for the ldm output sequences */
```

Comment:
minor detail : why a `rawSeq*` here, instead of a `rawSeqStore_t` ?

Reply:
I wanted to make it clear that no sequences are maintained between calls to the block compressor. Using a `rawSeqStore_t` leaves the possibility that `pos != 0` or `size != 0` at the beginning of the call. Then it might look like we are throwing out sequences.